### ISSUE TITLE
ngClass disappears with multiple conditions

### ISSUE THREAD

**Issue by KerimBayramoglu** [prettier/prettier#5422]
**Prettier 1.15.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAJgSwG4AIwA2AhgM4kC8AOiAGYQBOAtgLQDm9EArgA7UB8lKLmG5UmKN04xcMAJ7c4VEN1IkA7g3TVcAbSisAwsTIBdJcEEiruAOSYSzcdiIFM6G0kvWrACnpxWe3h6ADEGRgA6VjgYHxtIKBpMJgAFVQ16dwBKCLh6DnoSXAAyYq9vEX9AkmCwpiiYuISk1PTNGxyYLjAACzh0LNwAHyHyir8AoLy6yOjY+OgWxjSyDOyIrs5e-pKyoQrhKqnQ8Iie0gBRfIY4xntGIhhejqyx4QBfajf8YwpqOiYzASMA4BG0AMYBmgIIgBAAckRGIpqM1kss2pltNxiGA4D1Yeg8kpUUxcCpVpptAB6AT7YQYHA-VRKJwuNzMGhwfoAIyIYAA1toAFT6ACSNCU30Okxq01OcyaizRK3U7Ry5xIVwKcX8AEdOMl+h1dlLcEdZSd6gqFollRj1pttlo6SJ+N8AOp8nowDSaXDYPJwIIAQjGqCpWGwtKsDLwhGZ1FZrnQHK56F5AuFYolXxdVnNtVOGq1NxsdxIDyePWNpVNBblVsaNqWKrWHQ23T6zu8brzuE9vR9EE0CFw-IgSKEUEwMVwEAD-gQoZd4cj0dEEZwfBAABoQBBuDBMNASMhQERrmo0v4oKeUERsBA3LuQNz6Hz+TEAMoqMDiVjICCnBwHu3qMAQ7o9Jg8AkL+cBfggJDQTg0GyMg4BkC+4gkHkMApO+rAPMgNAuDhe4AFYkAAHgAQu+ArfoicAADLiHAxGkSBICUVRX7-gQcAAIqcBA8AcQQZHKBeOH0OhRD6JwxCyXu3D0OIMDum4MA9MgAAcAAMKkcDh7rvtw6GqXAMkBi+eoGv4+FEIRRDiZJOF3IB9DAXuSH6AJwmiexSAkRJXEwEQ3Kaeg2nIAATHuIJEJgrj6FCjBESgUDQOxe6cDhAAqEV3iFOHvO8QA)
```sh
# Options (if any):
angular
```

**Input:**
```jsx
<div class="form-group">
    <input type="password" [ngClass]="{
        'is-invalid':
          (registerForm.get('confirmPassword').errors &&
           registerForm.get('confirmPassword').touched) ||
          (registerForm.get('confirmPassword').touched &&
          registerForm.hasError('mismatch'))
      }"
      class="form-control" formControlName="confirmPassword" placeholder="confirm password" />
    <div class="invalid-feedback" *ngIf="
        registerForm.get('confirmPassword').hasError('required') &&
        registerForm.get('confirmPassword').touched
      ">
      Wachtwoord vereist!
    </div>
    <div class="invalid-feedback" *ngIf="
        registerForm.hasError('mismatch') &&
        registerForm.get('confirmPassword').touched
      ">
      Wachtwoorden komen niet overeen!
    </div>
  </div>
```

**Output:**
```jsx
<div class="form-group">
  <input
    type="password"
    [ngClass]="{
      '':
        (registerForm.get('confirmPassword').errors &&
          registerForm.get('confirmPassword').touched) ||
        (registerForm.get('confirmPassword').touched &&
          registerForm.hasError('mismatch'))
    }"
    class="form-control"
    formControlName="confirmPassword"
    placeholder="confirm password"
  />
  <div
    class="invalid-feedback"
    *ngIf="
      registerForm.get('confirmPassword').hasError('required') &&
      registerForm.get('confirmPassword').touched
    "
  >
    Wachtwoord vereist!
  </div>
  <div
    class="invalid-feedback"
    *ngIf="
      registerForm.hasError('mismatch') &&
      registerForm.get('confirmPassword').touched
    "
  >
    Wachtwoorden komen niet overeen!
  </div>
</div>
```

**Expected behavior:**

the class 'is-invalid' should not disappear when using OR '||'

**Comment by ikatyang** [comment]
Hi, this should be fixed by #5387, we plan to release a patch tomorrow.

**Prettier pr-5387**
[Playground link](https://deploy-preview-5387--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEAeAJgSwG4AIwA2AhgM4kC8AOiAGYQBOAtgLQDm9EArgA7UB8lKLmG5UmKN04xcMAJ7c4VEN1IkA7g3TVcAbSisAwsTIBdJcEEiruAOSYSzcdiIFM6G0kvWrACnpxWe3h6ADEGRgA6VjgYHxtIKBpMJgAFVQ16dwBKCLh6DnoSXAAyYq9vEX9AkmCwpiiYuISk1PTNGxyYLjAACzh0LNwAHyHyir8AoLy6yOjY+OgWxjSyDOyIrs5e-pKyoQrhKqnQ8Iie0gBRfIY4xntGIhhejqyx4QBfajf8YwpqOiYzASMA4BG0AMYBmgIIgBAAckRGIpqM1kss2pltNxiGA4D1Yeg8kpUUxcCpVpptAB6AT7YQYHA-VRKJwuNzMGhwfoAIyIYAA1toAFT6ACSNCU30Okxq01OcyaizRK3U7Ry5xIVwKcX8AEdOMl+h1dlLcEdZSd6gqFollRj1pttlo6SJ+N8AOp8nowDSaXDYPJwIIAQjGqCpWGwtKsDLwhGZ1FZrnQHK56F5AuFYolXxdVnNtVOGq1NxsdxIDyePWNpVNBblVsaNqWKrWHQ23T6zu8brzuE9vR9EE0CFw-IgSKEUEwMVwEAD-gQoZd4cj0dEEZwfBAABoQBBuDBMNASMhQERrmo0v4oKeUERsBA3LuQNz6Hz+TEAMoqMDiVjICCnBwHu3qMAQ7o9Jg8AkL+cBfggJDQTg0GyMg4BkC+4gkHkMApO+rAPMgNAuDhe4AFYkAAHgAQu+ArfoicAADLiHAxGkSBICUVRX7-gQcAAIqcBA8AcQQZHKBeOH0OhRD6JwxCyXu3D0OIMDum4MA9MgAAcAAMKkcDh7rvtw6GqXAMkBi+eoGv4+FEIRRDiZJOF3IB9DAXuSH6AJwmiexSAkRJXEwEQ3Kaeg2nIAATHuIJEJgrj6FCjBESgUDQOxe6cDhAAqEV3iFOHvO8QA)
```sh
--parser angular
```

**Input:**
```html
<div class="form-group">
    <input type="password" [ngClass]="{
        'is-invalid':
          (registerForm.get('confirmPassword').errors &&
           registerForm.get('confirmPassword').touched) ||
          (registerForm.get('confirmPassword').touched &&
          registerForm.hasError('mismatch'))
      }"
      class="form-control" formControlName="confirmPassword" placeholder="confirm password" />
    <div class="invalid-feedback" *ngIf="
        registerForm.get('confirmPassword').hasError('required') &&
        registerForm.get('confirmPassword').touched
      ">
      Wachtwoord vereist!
    </div>
    <div class="invalid-feedback" *ngIf="
        registerForm.hasError('mismatch') &&
        registerForm.get('confirmPassword').touched
      ">
      Wachtwoorden komen niet overeen!
    </div>
  </div>
```

**Output:**
```html
<div class="form-group">
  <input
    type="password"
    [ngClass]="{
      'is-invalid':
        (registerForm.get('confirmPassword').errors &&
          registerForm.get('confirmPassword').touched) ||
        (registerForm.get('confirmPassword').touched &&
          registerForm.hasError('mismatch'))
    }"
    class="form-control"
    formControlName="confirmPassword"
    placeholder="confirm password"
  />
  <div
    class="invalid-feedback"
    *ngIf="
      registerForm.get('confirmPassword').hasError('required') &&
      registerForm.get('confirmPassword').touched
    "
  >
    Wachtwoord vereist!
  </div>
  <div
    class="invalid-feedback"
    *ngIf="
      registerForm.hasError('mismatch') &&
      registerForm.get('confirmPassword').touched
    "
  >
    Wachtwoorden komen niet overeen!
  </div>
</div>

```

**Comment by ikatyang** [comment]
Duplicate of ##5367
